packages/safe-core-sdk/tests/execution.test.ts

```diff
@@ -197,21 +212,60 @@ describe('Transactions execution', () => {
         .expect(safeInitialBalance.toString())
         .to.be.eq(safeFinalBalance.add(BigNumber.from(tx.data.value).toString()))
     })
+  
+    it('should fail if a transaction that was rejected is executed', async () => {
```

Comment:
I found this test description hard to understand, I think something like `should fail if you try to execute rejected transaction`. Maybe I'm nitpicking, but I hard to read the test to understand what's happening

Reply:
What do you think about `should fail if the user tries to execute a transaction that was invalidated`?

Reply:
I don't like it because we don't really use `invalidated`, if you replace it with `rejected` then it sounds good to me. The description I suggested is even simpler IMO because it doesn't contain [passive voice](https://en.wikipedia.org/wiki/Passive_voice)